controllers/categories.js

```diff
@@ -0,0 +1,9 @@
+const { selectCategories } = require("../models/categories");
+
+exports.getCategories = (req, res, next) => {
+  selectCategories()
+    .then((categories) => {
+      res.status(200).send({ categories });
+    })
+    .catch(next);

```

Comment:
There is not currently an error handler to handle using this invocation of next. It would be acceptable to have a 500 error handler in `app.js` at this point.

Reply:
can i comment it out for now?

Reply:
i just removed all of them